### PR TITLE
Minor comma fixes

### PR DESCRIPTION
**Pull request by keflavich**
e.g., consistency and removal of a comma where there is only one clause

(don't merge quite yet - let me finish reading and I'll mark this as ready-to-merge)

### CODE REVIEW

File: APE14.rst [hunk 1]
```diff
@@ -330,7 +330,7 @@ Common UCD1+ names for physical types
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 As outlined above, the ``world_axis_physical_types`` attribute should include
-strings that follow the VO UCD1+ vocabulary for defining physical types. The
+strings that fullow the VO UCD1+ vocabulary for defining physical types. The
```
Comment (astrofrog) [inline APE14.rst]: Looks like you reverted a fix I added recently

**Comment by astrofrog** [comment]
We clearly need a grammar CI for APEs ;)

**Comment by keflavich** [comment]
some of these are pretty ambiguous, and there are cases where you've done things that are grammatically correct that aren't my style so I left them....

(I've been spending a lot of time on english.stackexchange.com recently looking at various wording conventions because I disagreed with some proof edits from apj)

**Comment by keflavich** [comment]
oh and this is ready to merge now if you agree

**Comment by astrofrog** [comment]
Cool, I agree with these fixes :) Let me know once you are done.

**Comment by astrofrog** [comment]
I merged this manually - thanks @keflavich!

**Pull request closed without merging**